tests/test_config.py

```diff
@@ -745,7 +745,8 @@ def merge_log_err(hass):
     await config_util.merge_packages_config(hass, config, packages)
     assert config["api"] == {
         "list_1": ["item_1", "item_2", "item_3"],
-        "list_2": ["item_1"],
+        "list_2": ["item_4"],
+        "list_3": [],
```

Comment:
The key to the fix is here.... person: being an empty list was discarded, like this list3